src/commands/preserve-auth0-cookies-once.js

```diff
@@ -12,7 +12,7 @@ Cypress.Commands.add('preserveAuth0CookiesOnce', () => {
     Cypress.env('auth0SessionCookieName'),
     // We assume that the cookie is never splitted into more than 10 parts
     ...Array(10)
-      .fill(1)
+      .fill(0)
       .map(
         (value, index) =>
           `${Cypress.env('auth0SessionCookieName')}.${value + index}`,
```

Comment:
```suggestion
    ...[...Array(10).keys()]
      .map(
        (value) => `${Cypress.env('auth0SessionCookieName')}.${value}`,
```
My tests create the auth0 cookies with the format `${Cypress.env('auth0SessionCookieName')}.${num}`, instead of `${Cypress.env('auth0SessionCookieName')}.0${num}`

Reply:
Your issue is fixed by switching the `fill(1)` to `fill(0)`. With that change, the array contains 10 `0` values indexed from `0` to `9`. The map function takes the index and then adds the corresponding value to it, which is always `0`, thus the return statement can be simplified to `${Cypress.env('auth0SessionCookieName')}.${index}`.